src/utl/src/LoggerCommon.h

```diff
@@ -43,9 +43,10 @@ void warn(utl::ToolId tool, int id, const char* msg);
 void error(utl::ToolId tool, int id, const char* msg);
 void critical(utl::ToolId tool, int id, const char* msg);
 void open_metrics(const char* metrics_filename);
+void close_metrics(const char* metrics_filename);
```

Comment:
When will we use this?

Reply:
Realistically, never, but I needed something to force the metrics to be flushed for CI, so I thought I could add it in case anyone wanted to record metrics in different files.
Without it the CI was not possible, so I think it's worth having something, but I'm open to suggestions